akka-http-core/src/main/scala/akka/http/scaladsl/model/ContentType.scala

```diff
@@ -11,13 +11,12 @@ import akka.http.javadsl.{ model ⇒ jm }
 import akka.http.impl.util.JavaMapping.Implicits._
 
 final case class ContentTypeRange(mediaRange: MediaRange, charsetRange: HttpCharsetRange) extends jm.ContentTypeRange with ValueRenderable {
-  def matches(contentType: jm.ContentType) = {
```

Comment:
Yeah, we could reconsider how we do things. But for now I guess we should keep it consistent with the rest of the code.